config.py

```diff
@@ -0,0 +1,5 @@
+import secrets
+
+SECRET_KEY = secrets.token_hex(16)
+MONGO_URI = 'mongodb://localhost:27017/your_database'
+VERSION = "1.0.0.0"
```

Comment:
good job, everytime you import config file it will allocate memory to assign these variables..
in order to improve youre code quality i would like a singleton design pattern which isnt really supported in python through regular means so in order to archive a singleton read about lru_cache decorator and singleton design pattern..

feel free to ask me questions if you get lost.

Reply:
also configuration file should be in an utill folder to keep it organized the root folder should only contain general files like readme and maybe license / main.py etc..

Reply:
What do we use it for? So that the function does not have to operate several times for the same inputs?

Reply:
Is it faster that way?

Reply:
singleton is a design pattern for saving unnessacary use of memory..

Reply:
yes i saw that i pushed the files again after the fix